docs/source/manual.rst

```diff
@@ -411,3 +411,78 @@ Usage example
      - N
      - NA
      - read alternative k-mer models from FILE
+
+
+phase-reads - (experimental)
+--------------------
+
+Overview
+"""""""""""""""""""""""
+
+Phase reads using heterozygous SNVs with respect to a reference genome 
+
+Input
+"""""""""""""""""""""""
+
+    * basecalled reads
+    * alignment information
+    * assembled genome
+    * variants (from nanopolish variants or from other sources eg. Illumina VCF)
+
+Usage example
+"""""""""""""""""""""""
+
+::
+
+   nanopolish phase-reads [OPTIONS] --reads reads.fa --bam alignments.bam --genome genome.fa variants.vcf
+
+.. list-table::
+   :widths: 20 10 20 50
+   :header-rows: 1
+
+   * - Argument name(s)
+     - Required
+     - Default value
+     - Description
+
+   * - ``-v``
+     - N
+     - NA
+     - write verbose output
+
+   * - ``-w, --window=STR``
+     - N
+     - NA
+     - Only phase reads in the window STR (format : ctg:start_id-end_id)
+
+   * - ``-r, --reads=FILE``
+     - Y
+     - NA
+     - the 2D ONT reads are in fasta FILE
```

Comment:
This should just be "ONT" reads, 2D reads are not required. I'll have to fix this in the usage of the program as well.